pyproject.toml

```diff
@@ -60,6 +60,8 @@ guardrails-api = ">=0.1.0a1,<0.2.0"
 mlflow = {version = "^2.0.1", optional = true}
 uvloop = {version = "^0.20.0", optional = true}
 semver = "^3.0.2"
+llama-index = {version = ">=0.11.0", optional = true}
+boto3 = {version = ">1, <2", optional = true}
```

Comment:
This boto3 dependency is just to help poetry resolve the version for when it's a transient dependency (it comes from both guardrails-api and mlflow).  Without this, trying to run a `poetry lock` will just spin.  It might resolve eventually but I had one run for over an hour without a resolution.